oak/server/rust/oak_runtime/src/node/grpc_server.rs

```diff
@@ -29,6 +29,40 @@ use oak_abi::{
 
 use crate::{pretty_name_for_thread, runtime::RuntimeProxy, Handle};
 
+use prometheus::{opts, register_histogram, register_int_counter, Histogram, IntCounter};
+
+struct GrpcMetrics {
+    request_duration: Histogram,
+    requests_total: IntCounter,
+    response_size: Histogram,
+}
```

Comment:
I think it would be more manageable to centralize the metrics definitions in a single object the metrics package, rather than spread it out. Also it will help to move away from the static initializer eventually. In fact, at that point, you may as well use a non-static Registry, but up to you if you want to leave that for another PR.